index.html

```diff
@@ -113,13 +113,435 @@ <h3 class="hero__form-title">Book Appointment</h3>
                         </select>
                         <label for="time">Time *</label>
                         <input type="time" name="time" id="time" placeholder="4:00 Avalible">
-                        <input  type="submit" value="book Appointment" class="btn" >
+                        <input type="submit" value="book Appointment" class="btn">
                     </form>
                 </div>
             </div>
         </header>
+        </div>
+        
         <!-- ========== End hero ========== -->
+        <!-- ========== Start leading ========== -->
+        <div class="leading">
```

Comment:
section element, should be the outter container, even if you will use containers, let them be divs inside.
body > main > section > div